### ISSUE TITLE
Add some more descriptions for ease of use

### ISSUE THREAD

**Issue by Jmang0** [Midataur/mathsender#31]
Some of this might just be Mr Finch, but some possible improvements.

### Mr Finch shared the password with us because he thought we needed it
- Maybe a description saying something like "This password is just for you to access the room"

### Mr Finch thought answers would disappear if he left the page
- Clarify that answers won't disappear when you go off a question

### It looks a bit blank when there aren't any questions
- Maybe a description like "When your teacher submits an answer, it'll pop up here"
- Similar thing when no answers have come though yet for teacher?

**Comment by ehne** [comment]
1. yeah, im going to add a bit of copy to emphasise that this is a secret password, don't share it with students.
2. hmm, that is an interesting one, should prolly ask around about that.
3. @Midataur and I were discussing adding a "blank-slate" type component to fill that empty space.

**Comment by jmang00** [comment]
### 1. Is Darcy gonna add something for this?

### 2. I've added some descriptions
#### Teacher side
- When there are no questions yet, saying:
> Once you submit a question, it'll appear below.

- When there are questions, saying:
> Click a question below to see answers from students.

- When there are no answers yet, saying:
> Answers will pop up here - tell your students to hurry up and submit!

#### Student side
- When there are no questions yet, saying:
> When your teacher submits a question, it'll pop up here.

- I HAD one when there were questions, but it seemed unnecessary

### 3. I was thinking maybe we could:
- change 'Go Back' to 'See all Questions'
- and/or add a back arrow icon

Thoughts?

**Comment by Midataur** [comment]
Yeah sounds pretty good to me. See all questions also sounds good

**Comment by ehne** [comment]
yeah, see all questions and the icon probably better indicates intent. and your little description texts fill the need for the "blank slate" thing I was talking about.
I still need to figure out a good way to tell them that the that the password shouldn't be shared. maybe just placeholder text that says something like: "keep this secret"?

**Comment by jmang00** [comment]
ok I'm gonna change it to 'See All Questions', darcy u can add an icon after u merge the icons branch.

**Comment by jmang00** [comment]
And I think the placeholder text might be the best bet for that.
Also I just realised there's nothing on the site telling the teacher to share the code. I'm not sure where to put that tho.

**Comment by Midataur** [comment]
Maybe a little pop up that points to the room code with a cookie to disable
it?

**Comment by ehne** [comment]
like, the MSAlert, but it shows on the first time loading that specific room code? 

and, @Jmang0 I will add icons in the icons branch, and ill just quickly put the placeholder text there.

**Comment by ehne** [comment]
<img width="1467" alt="Screen Shot 2020-09-11 at 7 36 59 pm" src="https://user-images.githubusercontent.com/7551957/92904387-2bfc0680-f466-11ea-9463-26d2beda1c47.png">

does this look good?

**Comment by jmang00** [comment]
Nice, maybe add at the bottom password too.

Maybe like MSAlert but near the room code button? Or more like the 'Please fill in this field' when you don't answer a password (in the sense that it points at the room code button)

**Comment by ehne** [comment]
im a little bit confused as to what you mean by `Or more like the 'Please fill in this field' when you don't answer a password (in the sense that it points at the room code button)`
